### PR TITLE
API Gateway v2 docs

### PR DESCRIPTION
**Pull request by tomekpapiernik**
**Description**

Changes proposed in this pull request:

- Added documentation for API Gateway v2
- Updated other docs that touch on this topic

**Related issue(s)**
#5305

### CODE REVIEW

File: resources/core/charts/docs/charts/content-ui/templates/docs-components-api-gateway-v2-cdt.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  labels:
+    cms.kyma-project.io/view-context: docs-ui
+    cms.kyma-project.io/group-name: components
+    cms.kyma-project.io/order: "11"
+  name: api-gateway
```
Comment (magicmatatjahu) [inline resources/core/charts/docs/charts/content-ui/templates/docs-components-api-gateway-v2-cdt.yaml]: `api-gateway-v2` :)

File: docs/api-gateway-v2/02-01-api-gateway.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: Architecture
+---
+
+This diagram illustrates the workflow that leads to exposing a service in Kyma:
+
+![service-exposure-flow](./assets/001-api-gateway-flow.svg)
+
+- **API Gateway Controller** is a component responsible for exposing services. The API Gateway Controller is an application deployed in the `kyma-system` Namespace, implemented according to the [Kubernetes Operator](https://coreos.com/blog/introducing-operators.html) principles. The API Gateway Controller listens for newly created custom resources (CR) that follow the set `apirule.gateway.kyma-project.io` CustomResourceDefinition (CRD), which describes the details of exposing services in Kyma.
```
Comment (piotrmsc) [inline docs/api-gateway-v2/02-01-api-gateway.md]: Follows k8s controller using kubebuilder https://book.kubebuilder.io/

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+<div tabs>
+  <details>
+  <summary>
+  Token with "read" scope
+  </summary>
+
+  1. Get the token:
+
+      ```
+      curl -ik -X POST "https://oauth2.$DOMAIN/oauth2/token" -H "Authorization: Basic $ENCODED_CREDENTIALS" -F "grant_type=client_credentials" -F "scope=read"
```
Comment (piotrmsc) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
      curl -ik -X POST "https://oauth2.$CLUSTER_DOMAIN/oauth2/token" -H "Authorization: Basic $ENCODED_CREDENTIALS" -F "grant_type=client_credentials" -F "scope=read"
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+  export CLIENT_SECRET="$(kubectl get secret -n $CLIENT_NAMESPACE $CLIENT_NAME -o jsonpath='{.data.client_secret}' | base64 --decode)"
+  ```
+
+4. Encode your client credentials and export them as an environment variable:
+
+  ```
+  export ENCODED_CREDENTIALS=$(echo -n "$CLIENT_ID:$CLIENT_SECRET" | base64)
+  ```
+
+5. Get tokens to interact with secured resources:
```
Comment (piotrmsc) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
5. Get tokens to interact with secured resources using client credentials flow:
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+      ```
+      export ACCESS_TOKEN_WRITE={ISSUED_WRITE_TOKEN}
+      ```
+
+   </details>
+</div>
+
+## Deploy, expose, and secure the sample resources
+
+Follow the instructions in the tabs to deploy an instance of the HttpBin service or a sample lambda function, expose them, and secure them with Oathkeeper rules.
```
Comment (piotrmsc) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
Follow the instructions in the tabs to deploy an instance of the HttpBin service or a sample lambda function, expose them, and secure them with Oauth2 scopes.
```

File: docs/api-gateway-v2/01-01-api-gateway.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+title: Overview
+---
+
+>**CAUTION:** This implementation replaces the API Gateway that based on the Api custom resource. The services you exposed and secured using the deprecated implementation require no action, as the API Gateway Controller co-exists with the API Controller in the cluster. Expose and secure new services and functions secured with OAuth2 using the `v2` implementation described in this documentation topic.
+
+To make your service accessible outside the Kyma cluster, expose it using the Kyma API Gateway Controller, which listens for the custom resource (CR) objects that follow the `apirule.gateway.kyma-project.io` CustomResourceDefinition (CRD). Creating a valid CR triggers the API Gateway Controller to create an Istio Virtual Service. Optionally, you can specify the **rules** attribute of the CR to secure the exposed service with Oathkeeper Access Rules.
```
Comment (piotrmsc) [inline docs/api-gateway-v2/01-01-api-gateway.md]:
```suggestion
To make your service accessible outside the Kyma cluster, expose it using the Kyma API Gateway Controller, which listens for the custom resource (CR) objects that follow the `apirules.gateway.kyma-project.io` CustomResourceDefinition (CRD). Creating a valid CR triggers the API Gateway Controller to create an Istio Virtual Service. Optionally, you can specify the **rules** attribute of the CR to secure the exposed service with Oathkeeper Access Rules.
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+Follow the instructions in the tabs to call the secured service or lambda functions using the tokens issued for the client you registered.
+
+<div tabs>
+
+  <details>
+  <summary>
+  Call secured endpoints of a service
+  </summary>
+
+1. Send a `GET` request with a token with the "read" scope to the HttpBin service:
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]: Is there a way to rephrase it to avoid so many `with`'s? (Also, lines 215 and 230.)

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+  curl -ik -X GET https://httpbin-proxy.$DOMAIN/headers -H "Authorization: Bearer $ACCESS_TOKEN_READ"
+  ```
+
+2. Send a `POST` request with a token with the "write" scope to the HttpBin's `/post` endpoint:
+
+  ```
+  curl -ik -X POST https://httpbin-proxy.$DOMAIN/post -d "test data" -H "Authorization: bearer $ACCESS_TOKEN_WRITE"
+  ```
+
+These calls return a code `200` response. If you call the service without a token, you get a code `401` response. If you call the service or its secured endpoint with a token with the wrong scope, you get the code `403` response.
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
These calls return the code `200` response. If you call the service without a token, you get the code `401` response. If you call the service or its secured endpoint with a token with the wrong scope, you get the code `403` response.
```
These codes are set and their meaning doesn't change, so I'd use `the`, like you did in the last sentence.

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+  Call the secured lambda function
+  </summary>
+
+Send a `GET` request with a token with the "read" scope to the lambda function:
+
+  ```
+  curl -ik https://lambda-proxy.$DOMAIN/lambda -H "Authorization: bearer $ACCESS_TOKEN_READ"
+  ```
+
+This call returns a code `200` response. If you call the lambda function without a token, you get a code `401` response. If you call the lambda function with a token with the wrong scope, you get the code `403` response.
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
This call returns the code `200` response. If you call the lambda function without a token, you get the code `401` response. If you call the lambda function with a token with the wrong scope, you get the code `403` response.
```

File: docs/api-gateway/01-01-api-gateway.md [hunk 10]
```diff
@@ -2,4 +2,6 @@
 title: Overview
 ---
 
+>**CAUTION:** This implementation of the API Gateway is **deprecated** until the entirety of its functionality is covered by the `v2` implementation. The services you exposed and secured using this implementation require no action, as the API Controller co-exists with the API Gateway Controller in the cluster. Expose and secure new services and functions secured with OAuth2 using the `v2` implementation. Read [this](/components/api-gateway-v2#overview-overview) documentation to learn more.
```
Comment (majakurcius) [inline docs/api-gateway/01-01-api-gateway.md]:
```suggestion
>**CAUTION:** This implementation of the API Gateway is **deprecated** until all of its functionality is covered by the `v2` implementation. The services you exposed and secured using this implementation require no action, as the API Controller co-exists with the API Gateway Controller in the cluster. Expose and secure new services and functions secured with OAuth2 using the `v2` implementation. Read [this](/components/api-gateway-v2#overview-overview) documentation to learn more.
```

File: docs/kyma/03-01-components.md [hunk 11]
```diff
@@ -45,7 +45,9 @@ The tracing in Kyma uses the [Jaeger](https://github.com/jaegertracing) distribu
 
 ## API Gateway
 
-The API Gateway aims to provide a set of functionalities which allow developers to expose, secure, and manage their APIs in an easy way. The main element of the API Gateway is the API Controller which exposes services in Kyma.
+The API Gateway aims to provide a set of functionalities which allow developers to expose, secure, and manage their APIs in an easy way. The main element of the API Gateway is the API Gateway Controller which exposes services in Kyma.
```
Comment (majakurcius) [inline docs/kyma/03-01-components.md]:
```suggestion
The API Gateway aims to provide a set of functionalities which allow developers to expose, secure, and manage their APIs in an easy way. The main element of the API Gateway is the API Gateway Controller, which exposes services in Kyma.
```

File: docs/kyma/03-01-components.md [hunk 12]
```diff
@@ -45,7 +45,9 @@ The tracing in Kyma uses the [Jaeger](https://github.com/jaegertracing) distribu
...
+
+>**CAUTION:** The API Gateway implementation that uses the API Controller and the Api custom resource is **deprecated** until the entirety of its functionality is covered by the `v2` implementation. The services you exposed and secured so far do not require any action as two implementations co-exist in the cluster. When you expose new services and functions secured with OAuth2, use the `v2` implementation. For more information, read [this](/components/api-gateway-v2#overview-overview) documentation.
```
Comment (majakurcius) [inline docs/kyma/03-01-components.md]:
```suggestion
>**CAUTION:** The API Gateway implementation that uses the API Controller and the Api custom resource is **deprecated** until all of its functionality is covered by the `v2` implementation. The services you exposed and secured so far do not require any action as two implementations co-exist in the cluster. When you expose new services and functions secured with OAuth2, use the `v2` implementation. For more information, read [this](/components/api-gateway-v2#overview-overview) documentation.
```

File: docs/api-gateway-v2/08-01-exposesecure.md (hunk 8 above)
Comment (tomekpapiernik) [reply docs/api-gateway-v2/08-01-exposesecure.md]: good point

File: docs/api-gateway/01-01-api-gateway.md (hunk 10 above)
Comment (tomekpapiernik) [reply docs/api-gateway/01-01-api-gateway.md]: 👍

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 13]
```diff
@@ -0,0 +1,239 @@
...
+Follow the instructions in the tabs to call the secured service or lambda functions using the tokens issued for the client you registered.
+
+<div tabs>
+
+  <details>
+  <summary>
+  Call secured endpoints of a service
+  </summary>
+
+1. Send a `GET` request with a token with the "read" scope to the HttpBin service:
```
Comment (tomekpapiernik) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
1. Send a `GET` request with a token that has the "read" scope to the HttpBin service:
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 14]
```diff
@@ -0,0 +1,239 @@
...
+
+  ```
+  curl -ik -X GET https://httpbin-proxy.$DOMAIN/headers -H "Authorization: Bearer $ACCESS_TOKEN_READ"
+  ```
+
+2. Send a `POST` request with a token with the "write" scope to the HttpBin's `/post` endpoint:
```
Comment (tomekpapiernik) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
2. Send a `POST` request with a token that has the "write" scope to the HttpBin's `/post` endpoint:
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 15]
```diff
@@ -0,0 +1,239 @@
...
+These calls return the code `200` response. If you call the service without a token, you get the code `401` response. If you call the service or its secured endpoint with a token with the wrong scope, you get the code `403` response.
+
+  </details>
+
+  <details>
+  <summary>
+  Call the secured lambda function
+  </summary>
+
+Send a `GET` request with a token with the "read" scope to the lambda function:
```
Comment (tomekpapiernik) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
Send a `GET` request with a token that has the "read" scope to the lambda function:
```

File: docs/api-gateway-v2/01-01-api-gateway.md [hunk 16]
```diff
@@ -0,0 +1,11 @@
+---
+title: Overview
+---
+
+>**CAUTION:** This implementation replaces the API Gateway that based on the Api custom resource. The services you exposed and secured using the deprecated implementation require no action, as the API Gateway Controller co-exists with the API Controller in the cluster. Expose and secure new services and functions secured with OAuth2 using the `v2` implementation described in this documentation topic.
```
Comment (tomekpapiernik) [inline docs/api-gateway-v2/01-01-api-gateway.md]:
```suggestion
>**CAUTION:** This implementation replaces the API Gateway that is based on the Api custom resource. The services you exposed and secured using the deprecated implementation require no action, as the API Gateway Controller co-exists with the API Controller in the cluster. Expose and secure new services and functions secured with OAuth2 using the `v2` implementation described in this documentation topic.
```

File: docs/api-gateway-v2/06-01-apirule.md [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+    host: foo.bar
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      mutators: []
+      accessStrategy:
+        - handler: oauth2_introspection
+          config:
+            required_scope: ["read"]
+
```
Comment (tomekpapiernik) [inline docs/api-gateway-v2/06-01-apirule.md]:
```suggestion
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 18]
```diff
@@ -0,0 +1,239 @@
...
+
+  - The domain of your cluster:
+
+    ```
+    export DOMAIN={CLUSTER_DOMAIN}
+    ```
+
+2. Create an OAuth2 client with `read` and `write` scopes. Run:
+
+  ```
```
Comment (tomekpapiernik) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```yaml
```

File: docs/api-gateway-v2/02-01-api-gateway.md [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+This diagram illustrates the workflow that leads to exposing a service in Kyma:
+
+![service-exposure-flow](./assets/001-api-gateway-flow.svg)
+
+- **API Gateway Controller** is a component responsible for exposing services. The API Gateway Controller is an application deployed in the `kyma-system` Namespace, implemented according to the [Kubernetes Controller](https://book.kubebuilder.io/) principles. The API Gateway Controller listens for newly created custom resources (CR) that follow the set `apirule.gateway.kyma-project.io` CustomResourceDefinition (CRD), which describes the details of exposing services in Kyma.
+
+- **Istio Virtual Service** specifies the services visible outside the cluster. The API Gateway Controller creates a Virtual Service for the hostname defined in the `apirule.gateway.kyma-project.io` CRD. The convention is to create a hostname using the name of the service as the subdomain, and the domain of the Kyma cluster. To learn more about the Istio Virtual Service concept, read this [Istio documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/).
+To get the list of Virtual Services in Kyma, run:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/02-01-api-gateway.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/02-01-api-gateway.md [hunk 20]
```diff
@@ -0,0 +1,24 @@
...
+  kubectl get virtualservices.networking.istio.io --all-namespaces
+  ```
+
+- **Oathkeeper Access Rule** allows operators to specify authentication requirements for a service. It is an optional resource, created only when the CR specifies the desired authentication method, the trusted token issuer, allowed methods and paths, and required scopes. To learn more about Oathkeeper Access Rules, read [this](https://www.ory.sh/docs/oryos.10/oathkeeper/api-access-rules) document.
+
+  To get the list of Oathkeeper Access Rules created in Kyma, run:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/02-01-api-gateway.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/06-01-apirule.md [hunk 21]
```diff
@@ -0,0 +1,72 @@
+---
+title: APIRule
+type: Custom Resource
+---
+
+The `apirule.gateway.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format the API Gateway Controller listens for. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
```
Comment (majakurcius) [inline docs/api-gateway-v2/06-01-apirule.md]:
```suggestion
```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 22]
```diff
@@ -0,0 +1,239 @@
...
+
+The tutorial comes with a sample HttpBin service deployment and a sample lambda function.
+
+## Register an OAuth2 client and get tokens
+
+1. Export these values as environment variables:
+
+  - The name of your client and the Secret which stores the client credentials:
+
+    ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
    ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 23]
```diff
@@ -0,0 +1,239 @@
...
+    export CLIENT_NAME={YOUR_CLIENT_NAME}
+    ```
+
+  - The Namespace in which you want to create the client and the Secret that stores its credentials:
+
+    ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
    ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 24]
```diff
@@ -0,0 +1,239 @@
...
+    export CLIENT_NAMESPACE={YOUR_CLIENT_NAMESPACE}
+    ```
+
+  - The domain of your cluster:
+
+    ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
    ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 25]
```diff
@@ -0,0 +1,239 @@
...
+    grantTypes:
+      - "client_credentials"
+    scope: "read write"
+    secretName: $CLIENT_NAME
+  EOF
+  ```
+
+3. Export the credentials of the created client as environment variables. Run:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 26]
```diff
@@ -0,0 +1,239 @@
...
+  export CLIENT_ID="$(kubectl get secret -n $CLIENT_NAMESPACE $CLIENT_NAME -o jsonpath='{.data.client_id}' | base64 --decode)"
+  export CLIENT_SECRET="$(kubectl get secret -n $CLIENT_NAMESPACE $CLIENT_NAME -o jsonpath='{.data.client_secret}' | base64 --decode)"
+  ```
+
+4. Encode your client credentials and export them as an environment variable:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 27]
```diff
@@ -0,0 +1,239 @@
...
+
+<div tabs>
+  <details>
+  <summary>
+  Token with "read" scope
+  </summary>
+
+  1. Get the token:
+
+      ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
      ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 28]
```diff
@@ -0,0 +1,239 @@
...
+      curl -ik -X POST "https://oauth2.$DOMAIN/oauth2/token" -H "Authorization: Basic $ENCODED_CREDENTIALS" -F "grant_type=client_credentials" -F "scope=read"
+      ```
+
+  2. Export the issued token as an environment variable:
+
+      ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
      ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 29]
```diff
@@ -0,0 +1,239 @@
...
+
+  </details>
+  <details>
+  <summary>
+  Token with "write" scope
+  </summary>
+
+  1. Get the token:
+
+      ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
      ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 30]
```diff
@@ -0,0 +1,239 @@
...
+      curl -ik -X POST "https://oauth2.$DOMAIN/oauth2/token" -H "Authorization: Basic $ENCODED_CREDENTIALS" -F "grant_type=client_credentials" -F "scope=write"
+      ```
+
+  2. Export the issued token as an environment variable:
+
+      ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
      ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 31]
```diff
@@ -0,0 +1,239 @@
...
+<div tabs>
+
+  <details>
+  <summary>
+  HttpBin - secure endpoints of a service
+  </summary>
+
+1. Deploy an instance of the HttpBin service:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 32]
```diff
@@ -0,0 +1,239 @@
...
+  kubectl apply -f https://raw.githubusercontent.com/istio/istio/master/samples/httpbin/httpbin.yaml
+  ```
+
+2. Expose the service and secure it by creating an APIRule CR:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 33]
```diff
@@ -0,0 +1,239 @@
...
+  </details>
+
+  <details>
+  <summary>
+  Secure a lambda function
+  </summary>
+
+1. Create a lambda function using the [supplied code](./assets/lambda.yaml):
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 34]
```diff
@@ -0,0 +1,239 @@
...
+  kubectl apply -f ./assets/lambda.yaml
+  ```
+
+2. Expose the lambda function and secure it by creating an APIRule CR:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 35]
```diff
@@ -0,0 +1,239 @@
...
+<div tabs>
+
+  <details>
+  <summary>
+  Call secured endpoints of a service
+  </summary>
+
+1. Send a `GET` request with a token that has the "read" scope to the HttpBin service:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 36]
```diff
@@ -0,0 +1,239 @@
...
+  curl -ik -X GET https://httpbin-proxy.$DOMAIN/headers -H "Authorization: Bearer $ACCESS_TOKEN_READ"
+  ```
+
+2. Send a `POST` request with a token that has the "write" scope to the HttpBin's `/post` endpoint:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```shell
```

File: docs/api-gateway-v2/08-01-exposesecure.md [hunk 37]
```diff
@@ -0,0 +1,239 @@
...
+  </details>
+
+  <details>
+  <summary>
+  Call the secured lambda function
+  </summary>
+
+Send a `GET` request with a token that has the "read" scope to the lambda function:
+
+  ```
```
Comment (majakurcius) [inline docs/api-gateway-v2/08-01-exposesecure.md]:
```suggestion
  ```shell
```
